src/components/Tokens/TokenDetails/Skeleton.tsx

```diff
@@ -97,31 +97,32 @@ const TokenLogoBubble = styled(DetailBubble)`
   border-radius: 50%;
 `
 const TitleBubble = styled(DetailBubble)`
-  width: 140px;
+  width: 137px;
```

Comment:
should be a `<px value> % 4 === 0` value

Reply:
sg done